client/components/date-range/style.scss

```diff
@@ -248,40 +255,36 @@ $date-range-mobile-layout-switch: $break-small;
 			&:hover,
 			&:focus {
 				color: var(--color-text-inverted);
-				background-color: var(--date-range-picker-highlight-color);
+				background-color: var(--color-primary);
 			}
 		}
 	}
 
-	.DayPicker-Day--range .date-picker__day {
-		color: var(--color-text);
-		background-color: var(--date-range-picker-highlight-color);
-	}
-
-	.DayPicker-Day--start.DayPicker-Day--range-start,
-	.DayPicker-Day--end.DayPicker-Day--range-end {
-		& .date-picker__day {
+	.DayPicker-Day--range,
```

Comment:
are these changes intended?

Reply:
hmmm maybe my rebase got a bit out of control here. Those look like your changes from yesterday with the range styling, right?

Reply:
Looks like a rebase issue here yes. If you take a look at the highlight colors, is it still working?